File: taller/SergioDeIscar-taller-java/SergioDeIscar-taller-java/src/main/java/org/example/models/Electricista.java
package org.example.models;

public class Electricista extends Trabajador{
    public Electricista(String nombre, int experiencia, int horasDiarias) {
        super(nombre, experiencia, horasDiarias);
    }

    int salario = 1800;

    @Override
    public void comer() {
        System.out.println("El trabajador chapista " + nombre + " come.");
    }

    @Override
    public int getSalario() {
        return salario;
    }

    @Override
    public String toString() {
        return "Electricista -> Nombre: " + nombre + ", Experiencia: " + experiencia + ", Horas diarias: " + horasDiarias + ", Salario: " + salario;
    }
}

File: taller/SergioDeIScar-taller-kotlin/SergioDeIscar-taller-kotlin/src/main/kotlin/models/Electricista.kt
package taller.models

open class Electricista(nombre: String, experiencia: Int, horasDiarias: Int): Trabajador(nombre, experiencia, horasDiarias) {
    override val salario: Int = 1800
    override fun comer() {
        println("El trabajador electricista $nombre come.")
    }

    override fun toString(): String {
        return "Electricista -> Nombre: $nombre, Experiencia: $experiencia, Horas diarias: $horasDiarias, Salario: $salario"
    }
}